roles/hdfs-standalone/tasks/hdfs.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Setup basic directories on HDFS
+  run_once: true
+  sudo_user: hdfs
+  sudo: true
+  shell: source /etc/profile.d/hadoop.sh; if ! hdfs dfs -test -d /tmp; then hdfs dfs -mkdir /tmp; fi; hdfs dfs -chmod 777 /tmp; if ! hdfs dfs -test -d /user; then hdfs dfs -mkdir -p /user; fi; if ! hdfs dfs -test -d /apps; then hdfs dfs -mkdir -p /apps; fi
```

Comment:
again, what happens if one of these commands fails halfway? Since it's using `;`, will it blithely continue to run?